docs/includes/quickstart-yo-start-server-excel.md

```diff
@@ -2,7 +2,7 @@
 Complete the following steps to start the local web server and sideload your add-in.
 
 > [!NOTE]
-> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides.
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides. You may also have to run your command prompt or terminal as an administrator for the changes to be made.
```

Comment:
Should this be changed to the new include too? I think includes and can include includes.